.vsts-ci-official.yml

```diff
@@ -0,0 +1,183 @@
+variables:
+  - name: _TeamName
+    value: DotNetCore
+  - name: _DotNetArtifactsCategory
+    value: .NETCore
+  - name: Codeql.Enabled
+    value: true
+
+# CI and PR triggers
+trigger:
+  batch: true
+  branches:
+    include:
+    - main
+
+pr:
+  autoCancel: false
+  branches:
+    include:
+    - '*'
+
+resources:
+  repositories:
+  - repository: 1ESPipelineTemplates
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
+
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
+  parameters:
+    pool:
+      name: NetCore1ESPool-Svc-Internal
+      image: 1es-windows-2022-pt
+      os: windows
+    customBuildTags:
+    - ES365AIMigrationTooling
+    stages:
+    - stage: build
+      displayName: Build and Test
+      jobs:
+      - template: /eng/common/templates-official/jobs/jobs.yml@self
+        parameters:
+          enableMicrobuild: true
+          enablePublishBuildArtifacts: true
+          enablePublishUsingPipelines: true
+          enablePublishTestResults: true
+          enablePublishBuildAssets: true
+          enableTelemetry: true
+          enableSourceBuild: true
+          helixRepo: dotnet/command-line-api
+          timeoutInMinutes: 180 # increase timeout since BAR publishing might wait a long time
+          jobs:
+          - job: Windows
+            pool:
+              # For public or PR jobs, use the hosted pool.  For internal jobs use the internal pool.
+              # Will eventually change this to two BYOC pools.
+              ${{ if ne(variables['System.TeamProject'], 'internal') }}:
+                name: NetCore-Public
+                demands: ImageOverride -equals windows.vs2022.amd64.open
+              ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+                name: NetCore1ESPool-Internal
+                demands: ImageOverride -equals windows.vs2022.amd64
+            variables:
+            # Only enable publishing in official builds.
+            - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
```

Comment:
`ne(variables['System.TeamProject'], 'public')` is always true for this official-only pipeline